### PR TITLE
Doc Rita and settings

### PR DESCRIPTION
**Pull request by jkilpatr**
This commit adds documentaiton block paragraphs with the goal of describing
the functions of each file at the top of the file.

### CODE REVIEW

File: rita/src/exit.rs [hunk 1]
```diff
@@ -1,3 +1,14 @@
+//! This is the main source file for the Rita exit binary, by 'exit' we mean 'a vpn server, not a
+//! mesh router out in the field'.
+//!
+//! All meshing and billing functionaltiy is contained in rita_common and is common to both rita and
+//! rita_exit. The major difference is billing and connection code for the 'exit', the mandatory
```
Comment (kingoflolz) [inline rita/src/exit.rs]: should be traffic counting that is different, to distinguish from "billing functionality" which you say is common

File: rita/src/rita_client/traffic_watcher/mod.rs [hunk 2]
```diff
@@ -1,3 +1,11 @@
+//! Traffic watcher monitors system traffic by interfacing with KernelInterface to create and check
+//! ebtables counters on each per hop tunnel (the WireGuard tunnel between two devices). These counts
```
Comment (kingoflolz) [inline rita/src/rita_client/traffic_watcher/mod.rs]: we don't use ebtables anymore, we use iptables and ipset

File: rita/src/rita_common/traffic_watcher/mod.rs [hunk 3]
```diff
@@ -1,3 +1,7 @@
+//! Traffic watcher monitors system traffic by interfacing with KernelInterface to create and check
+//! ebtables counters on each per hop tunnel (the WireGuard tunnel between two devices). These counts
```
Comment (kingoflolz) [inline rita/src/rita_common/traffic_watcher/mod.rs]: same here, iptables and ipset

File: rita/src/rita_exit/db_client/mod.rs [hunk 4]
```diff
@@ -1,3 +1,7 @@
+//! DB client is essentially a layer over Diesel which is itself a layer over a sqllite database
+//! at some point we will need to add multi-database support such that we can scale exits
+//! horizontally
+
```
Comment (kingoflolz) [inline rita/src/rita_exit/db_client/mod.rs]: also contains a lot of the sign up logic and stuff (which it probs should not)

File: rita/src/rita_exit/network_endpoints/mod.rs [hunk 5]
```diff
@@ -1,3 +1,6 @@
+//! Network endpoints for rita-exit there are not dashboard or local infromational endpoints
```
Comment (kingoflolz) [inline rita/src/rita_exit/network_endpoints/mod.rs]: "that" instead of "there"?

File: rita/src/rita_exit/traffic_watcher/mod.rs [hunk 6]
```diff
@@ -1,3 +1,11 @@
+//! Traffic watcher monitors system traffic by interfacing with KernelInterface to create and check
+//! ebtables counters on each per hop tunnel (the WireGuard tunnel between two devices). These counts
```
Comment (kingoflolz) [inline rita/src/rita_exit/traffic_watcher/mod.rs]: same here, iptables and ipset

File: rita/src/rita_client/traffic_watcher/mod.rs (hunk 2 above)
Comment (jkilpatr) [reply rita/src/rita_client/traffic_watcher/mod.rs]: why switch? specifically the iptables lock is a scaling issue.

**Pull request merged**